memiavl/db.go

```diff
@@ -203,6 +203,13 @@ func Load(dir string, opts Options) (*DB, error) {
 		}
 	}
 
+	snapshotDir := snapshotName(db.lastCommitInfo.Version)
+	tmpDir := snapshotDir + "-tmp"
+	path = filepath.Join(db.dir, tmpDir)
+	if err := os.Remove(path); err != nil {
```

Comment:
I think we need to `RemoveAll` here

Reply:
yea, but not sure if del with -tmp suffix will be heavy

Reply:
should be alright, this is just the startup.